_documentation/user/flags/ricochet.md

```diff
@@ -5,4 +5,12 @@ type: Good
 summary: Shots bounce off walls. Don't shoot yourself!
 ---
 
-...
+The Ricochet (R) flag allows the player to fire shots that can bounce off of the objects they hit. The direction of the bounce is determined by the angle under which the platform is hit. An exception are objects with the attributes `shootthrough` or `passable`, where the result is the bullet passing through the object.
+
+A lot of maps utilize the default ricochet option to already allow the effect of Ricochet without the tanks being required to hold the specific flag for it. The Ricochet flag holds meaning and effect on maps that do not utilize said option.
```

Comment:
Maybe it's just an evolving pet peeve, but "a lot of maps" is irrelevant to the description of the flag.
```suggestion
The Ricochet flag has no value on maps that enable ricochet. For maps where ricochet is not enabled, the Ricochet flag offers the holder the advantage of banking shots.
```

Reply:
@jwmelto agreed, will change.